App/Redux/UIRedux.ts

```diff
@@ -63,14 +64,8 @@ const actions = {
   dismissImagePickerError: createAction('DISMISS_IMAGE_PICKER_ERROR', resolve => {
     return () => resolve()
   }),
-  refreshMessagesRequest: createAction('REFRESH_MESSAGES_REQUEST', resolve => {
-    return (hidden?: boolean) => resolve({hidden})
-  }),
-  refreshMessagesSuccess: createAction('REFRESH_MESSAGES_SUCCESS', resolve => {
-    return (timestamp: number) => resolve({ timestamp })
-  }),
-  refreshMessagesFailure: createAction('REFRESH_MESSAGES_FAILURE', resolve => {
-    return (error: Error) => resolve({ error })
+  routeDeepLinkRequest: createAction('ROUTE_DEEP_LINK_REQUEST', resolve => {
```

Comment:
This is duplicated here and in `TriggersRedux`

Reply:
doh, fixed. no idea why i did that